README.md

```diff
@@ -22,25 +22,32 @@ Google AdMob Integration:
 
 ## Installation
 
-1. [CocoaPods](https://cocoapods.org) is a dependency manager for Cocoa projects. For usage and installation instructions, visit their website.
- Before running the project you have to install the cocoapods dependecy to the project by running bellow command using terminal in root folder of desired Sample App (Swift or Objective C)
+_**CocoaPods** is a dependency manager for Cocoa projects and it's required for this project. For usage and installation instructions, visit [their official documentation](https://guides.cocoapods.org/)._
 
-```bash
-pod install
-```
+1. Install the cocoapods dependecy to the project by running bellow command using terminal in root folder of desired language:
+ 	 - [Swift](https://github.com/Ogury/ogury-sdk-sampleapp-ios/tree/master/Swift "Swift")
+	  - [Objective C](https://github.com/Ogury/ogury-sdk-sampleapp-ios/tree/master/Swift "Objective C")

```

Comment:
remark (blocking) : https://github.com/Ogury/ogury-sdk-sampleapp-ios/tree/master/Swift "Objective C" are you sure